mitodl/mitxpro-theme
# style: self hosted logo url & updated PNG logo

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/3203

### Description (What does it do?)

- config based email logo URL
- new PNG logo 5x for better quality over browser zooming


### How can this be tested?
Follow the same setup steps as mentioned in https://github.com/mitodl/mitx-theme/pull/142 but with the exception that you should see the xPRO logo.

### Additional Context
- We recently added PNG based logo, That turned out to be a low quality export from Figma, We now add a 5x PNG logo so that it stay as normal as possible on browser zoom. However, With our new production deployment the logo will automatically be replaced with and SVG format.
